New Mindful Messaging docs (#369)
cyclr/docs: pages/connector-authentication/mindful-messaging.md

---
title: Mindful Messaging Authentication
sidebar: cyclr_sidebar
permalink: mindful-messaging-connector
tags: [connector]
---

## Mindful Messaging setup

To install the Mindful Messaging connector within a Cyclr account you need an **API token** associated with your Mindful account.

<a href="get-api-token">

### Get API token

> You can find Mindful's documentation on how to get an API token for your Mindful account [here](https://help.vhtcx.com/mindful-messaging-configuration-settings/messaging-api-token?from_search=102822894). If you need help to obtain this, contact the Mindful support team.

Reviewer: ```suggestion
You can find Mindful's documentation on how to get an API token for your Mindful account [here](https://help.vhtcx.com/mindful-messaging-configuration-settings/messaging-api-token?from_search=102822894). If you need help to obtain the token, contact the Mindful support team.
```

## Cyclr account setup

> When installing the Mindful Messaging connector within an account you will be asked for the following:

Reviewer: ```suggestion
Cyclr asks you for the below values when you install the Mindful Messaging connector into an account:
```
